docs/UserGuide.md

```diff
@@ -190,16 +196,16 @@ Finds passengers whose names contain any of the given keywords.
 
 **:information_source: Notes about the find command:**<br>
 
-* The search is case-insensitive. e.g `hans` will match `Hans`.
-* Only full words will be matched e.g. `Han` will not match `Hans`.
+* The search is case-insensitive. e.g., `hans` will match `Hans`.
+* Only full words will be matched e.g., `Han` will not match `Hans`.
 * Only **one type of prefix** can be specified.
 * More than 1 keyword can be specified for a single prefix.
-* Keywords separated by space will require both keywords to be matched in whole and cannot be broken apart.
-  e.g. `Hans Yang` will only return `Gruber Hans Yang` instead of `Bo Yang` and `Hans Gruber Yang`.
+* Keywords separated by space will match all as a phrase rather than individual words.
+  e.g., `Hans Yang` will only return `Gruber Hans Yang` instead of `Bo Yang` and `Hans Gruber Yang`.
 * Prefixes for searching name `n/`, address `a/`, tag `tag/`, phone number `p/`, price `pr/`, day `d/` and time `t/`.
-* Searching with prefixes such as `n/` and `a/` which are anticipated to have multiple words separated by spaces will have multiple spaces shortened to one. Refer to 4th example for clarification.
+* Searching with prefixes such as `n/` and `a/` which are anticipated to have multiple words separated by spaces will have multiple spaces shortened to one. Refer to the 4th example for clarification.
 * `all` prefix for searching across names, addresses, tags and phone numbers quickly.
-* However, if any or all of the words are invalid arguments, searching with `all` will not tell you it is invalid. It will only show that no passengers with those arguments are found.
+	* However, if any or all of the words after the `all` prefix are invalid words, GME will not inform you as such. It will instead show that no passengers with those words are found.
```

Comment:
this sentence could perhaps be rephrased.
"if any or all of the words after the `all` prefix are invalid words"
since the subject is already 'words', re-stating them again as 'invalid words' is not necessary.

then again there's the part 'any or all' which makes too many clauses in the sentence. perhaps could pick one of them, because 'any' or 'all' kind of imply the same thing, unless i may have gotten this wrong.

TLDR my suggestion is to remove the second word 'word', and remove the word all 'a' -> "if any of the words after the `all` prefix are invalid"